### PR TITLE
Added new Blog post On German webpage

### PR DESCRIPTION
**Pull request by lokeshmittaldev**
Fixes: #603

### CODE REVIEW

**Comment by Poorvasisodia** [comment]
@lokeshmittaldev - An addition. We need to update an image also for this blog post just like the way we have images for other blogs as well. Please find it here and update along with this post as a Title image and not  in the main content page.
male-worker-factory

**Comment by Poorvasisodia** [comment]
@lokeshmittaldev - just for your reference title of the blog is - KI in Mittelstand

**Comment by Poorvasisodia** [comment]
@lokeshmittaldev - Please modify „
1. German Mittelstand“- to -"German Mittelstand".
2. First paragraph-its written in Italic in the word document I shared with you. It should be the similar way in the blog as well.
3. Remove the contact slides you have added in the ending. Instead please add- Folhalten Sie uns mehr auf LinkedIn, um auf dem Laufenden zu bleiben und fühlen Sie sich willkommen, uns bei Fragen zu kontaktieren.
4. Here, LinkedIn and kontaktieren-word should have hyperlinks of Company LinkedIn page and contact us form page.

**Comment by Poorvasisodia** [comment]
@lokeshmittaldev -New changes in the blog post.

1.I have attached another word document here. You need to replace with this.
[20210215.KI.im.Mittelstand_Version2.docx](https://github.com/tvarit-foggy/tvarit.com/files/6035588/20210215.KI.im.Mittelstand_Version2.docx)
2. Change the title to - KI im Mittelstand.
3. Remove the last line of LinkedIn and contact that we have added separately .
4. Do not add anything further apart from this page.

**Comment by Poorvasisodia** [comment]
@lokeshmittaldev KI im Mittelstand

File: de/blogs/index.php [hunk 1]
```diff
@@ -322,6 +322,20 @@ class="fas fa-search"></i></a>
...
+                        <div class="col-md-6 mix Topic">
+                            <div class="blogBox">
+                                <div class="blogPic">
+                                    <div class="industiesText">
+                                        Cross industry
+                                    </div>
+                                    <img src="img/ki-in-mittelstand.jpg" alt="Oil Gas" class="img-fluid">
+                                </div>
+                                <div class="content">
+                                    <a href="KI-im-Mittelstand" class="title">KI im Mittelstand</a>
```
Comment (starkblaze01) [inline de/blogs/index.php:334]: Replace it with
`
<a href="KI-im-Mittelstand" class="title" style="text-transform:none;">KI im Mittelstand</a>
`
try adding !important if it doesn't work.

File: de/blogs/ki-in-mittelstand.php [hunk 2]
```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
```
Comment (Sumeshks29) [inline de/blogs/ki-in-mittelstand.php:1]: @lokeshmittaldev  This is a duplicate file. remove this file if not in use

Comment (Sumeshks29) [reply de/blogs/ki-in-mittelstand.php:1]: image name is also incorrect. please fix that.

File: de/blogs/ki-im-mittelstand.php [hunk 3]
```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
```
Comment (Sumeshks29) [inline de/blogs/ki-im-mittelstand.php:1]: change the file name to KI-im-Mittelstand.php

**Pull request merged**